trade-finance-manager-ui/server/types/view-models/deal-cancellation/effective-from-date-view-model.ts

```diff
@@ -0,0 +1,16 @@
+import { BaseViewModel } from '../base-view-model';
+import { ErrorSummaryViewModel } from '../error-summary-view-model';
+
+export type EffectiveFromDateErrorsViewModel = {
+  summary: ErrorSummaryViewModel[];
+  effectiveFromDateError: { message: string; fields: string[] };
+};
+
+export type EffectiveFromDateViewModel = BaseViewModel & {
+  ukefDealId: string;
+  dealId: string;
+  day?: string;
+  month?: string;
+  year?: string;
+  errors?: EffectiveFromDateErrorsViewModel | null;
```

Comment:
Would this work?


`errors?: EffectiveFromDateErrorsViewModel`

Reply:
Ah yes, I think so thank you! Have changed